docs/1.getting-started/11.testing.md

```diff
@@ -333,7 +333,7 @@ registerEndpoint("/test/", {
 })
 ```
 
-> **Note**: If your requests in a component go to external API, you can use `baseURL` and then make it empty using Nuxt Environment Config (`$test`) so all your requests will go to Nitro server.
+> **Note**: If your component sends requests to an external API, you can configure a `baseURL` for [fetch](/docs/api/utils/dollarfetch) operations based on environment variables. By setting `baseURL` to an empty string through [Nuxt's Environment Configuration](/docs/guide/directory-structure/env#custom-file) (using .env.test), you ensure that all requests are directed to the Nitro server.
```

Comment:
Previously this was advising using `$test`, for example:

```ts
export default defineNuxtConfig({
  $test: {
    runtimeConfig: {
      apiBase: ''
    }
  }
})
```

This might be more straightforward than `.env.test`

Reply:
I was not able to find documentation for that, I agree it's more straightforwards but as far as I can tell it's not documented